dodola/core.py

```diff
@@ -162,7 +162,7 @@ def train_analogdownscaling(
     return aiqpd
 
 
-def adjust_analogdownscaling_year(simulation, aiqpd, year, variable):
+def adjust_analogdownscaling_year(simulation, aiqpd, variable):
```

Comment:
```suggestion
def adjust_analogdownscaling(simulation, aiqpd, variable):
```

Because we're no longer operating on individual years.

...The function sounds kinda weird but I can live with this for now if you can. I think it'd be okay to rename this to `apply_analogdownscaling` if that sounds more natural to you.

Reply:
Totally agree - updated as you suggested.